doit/dependency.py

```diff
@@ -693,6 +693,12 @@ def get_status(self, task, tasks_dict, get_log=False):
 class UptodateCalculator(object):
     """Base class for 'uptodate' that need access to all tasks
     """
+    
```

Comment:
is the pickle log complete? I am still missing the upper part. I know `result_dep` is instance is not picklable. What I dont understand is why it is even trying to pickle that. See that `Task.pickle_safe_dict` already removes the `uptodate` attribute from the task.

An even more interesting question is why only py2.7 is affected!